src/legacy/core_plugins/vis_type_vislib/public/vislib/helpers/point_series/_init_y_axis.ts

```diff
@@ -28,12 +30,7 @@ export function initYAxis(chart) {
 
   const z = chart.aspects.series;
   if (z) {
-    if (Array.isArray(z)) {
```

Comment:
`z` is always be an [array](https://github.com/elastic/kibana/blob/aab3dffefd1807a191d9d466a79eb079677f2277/src/plugins/visualizations/public/legacy/build_pipeline.ts#L457).